mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -12,6 +12,8 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 	METHOD method_35226 empty ()Lnet/minecraft/class_1856;
 	METHOD method_47374 (Lnet/minecraft/class_1856$class_1859;)Ljava/util/stream/Stream;
 		ARG 0 entry
+	METHOD method_52177 fromJsonAirAllowed (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_1856;
```

Comment:
Yes, I agree 👍

Reply:
I don't like things with the same signature but different behavior from old one being named the same. Previously this disallowed air, and now this allows air. By ensuring that compile fails, they can examine their code to see which one is better.

(also @modmuss50, I believe the ingredient API port used this method, but you should've used the boolean one. See, this is the kind of thing I am trying to prevent here.)

Reply:
While I see your reasoning apple, I do feel it should be an overload. Having a bad name for the sake of temporary backwards-compat (or rather backwards-incompat?) is something I'd rather avoid.

Reply:
```suggestion
	METHOD method_52177 fromJson (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_1856;
```